app/controllers/bookingsync_portal/admin/rentals_controller.rb

```diff
@@ -18,6 +17,26 @@ def show
 
       private
 
+      def not_connected_rentals
+        BookingsyncPortal.not_connected_rentals || Proc.new {
```

Comment:
why proc? I think lambda accepting `current_account` as argument will be cleaner